doc/api/quic.md

```diff
@@ -842,6 +842,18 @@ added: REPLACEME
 
 A `BigInt` representing the length of time the `QuicSession` was active.
 
+#### quicsession.earlyData
```

Comment:
Can we name this (and the option) something like `hasEarlyData`, `allowsEarlyData`, or similar, so that it sounds more boolean-y?